server/common/utils/version.py

```diff
@@ -0,0 +1,2 @@
+__version__ = "0.16.7"
+display_version = "cellxgene v" + __version__
```

Comment:
It would be acceptable to have this file all that way up in the `backend` dir.   I wouldn't consider it a `util` module.

Reply:
Make this change before committing